# Exceção personalizada para usuários inválidos
class UsuarioInvalidoError(Exception):
    pass
